### ISSUE TITLE
rgdal package

### ISSUE THREAD

**Issue by Han-Su22** [PIK-LPJmL/LandInG#5]
The maintenance of rgdal package has been terminated since Oct 2023. I could still install the package via:

install.packages("rgdal", repos="http://r-forge.r-project.org/")

Will you consider replacing rgdal package in the future?

**Comment by sostberg** [comment]
Yes, replacing the `rgdal` package is planned for a future update of LandInG. There is also a plan to replace the `raster` package by `terra`, which is the successor of `raster`.

**Comment by sostberg** [comment]
Update to version 1.1.0 has finally been released, see 268018b591d9bcdd5bd8645a5079f178477db559.
